File: src/com/funivan/phpstorm/refactoring/FindMagicMethods/MagicElementVisitor.java
package com.funivan.phpstorm.refactoring.FindMagicMethods;

/**
 * @author Ivan Shcherbak <alotofall@gmail.com>
 */
public interface MagicElementVisitor {

    ElementResultCollector getResultCollector();

    void setResultCollector(ElementResultCollector filter);

}

File: src/com/funivan/phpstorm/refactoring/findMagicMethods/MagicElementVisitor.kt
package com.funivan.phpstorm.refactoring.findMagicMethods

import com.funivan.phpstorm.refactoring.findMagicMethods.results.ResultCollectorInterface

/**
 * @author Ivan Shcherbak alotofall@gmail.com>
 */
interface MagicElementVisitor {
    var collector: ResultCollectorInterface
}
